backend/app/routers/auth.py

```
@@ -14,11 +14,11 @@ async def test_endpoint():
 async def login_user(user: schemas.UserLogin):
     """Login user using Supabase authentication"""
     try:
-        supabase: Client = await get_supabase_anon_client()
+        supabase: AsyncClient = await get_supabase_anon_client()
         
         if not supabase:
             raise HTTPException(
-                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                status_code=status.HTTP_500_INTERNAL_SERVER_.INTERNAL_SERVER_ERROR,
                 detail="Supabase not configured. Please set SUPABASE_URL and SUPABASE_KEY environment variables."
             )
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)